### PR TITLE
changes aggregated text to total

### PR DESCRIPTION
**Pull request by turnerniles**
addresses #69. I see text such as, 'Totals', 'Grand Total' in other projects. This PR change the row and column header text to 'Totals'. What do you think @pat310?

### CODE REVIEW

File: test/logic/tableCreator.js [hunk 1]
```diff
@@ -246,9 +246,10 @@ export default () => {
         () => {
           const tableResults = tableCreator(data, ['borough'], [], 'age',
               'count');
+          console.log('table', tableResults.table[0])
```
Comment (pat310) [inline test/logic/tableCreator.js]: remove

**Comment by pat310** [comment]
@turnerniles ☝️

File: test/logic/tableCreator.js (hunk 1 above)
Comment (turnerniles) [reply test/logic/tableCreator.js]: good spotting!

**Comment by turnerniles** [comment]
@pat310 Updated the README. 'Aggregated' or 'aggregated' in both places could also work.

**Pull request merged**